### DISCUSSION TITLE
Не увеличивает размер матрицы!

### DISCUSSION THREAD

**Discussion by kovalserg** [vvip-68/LedPanelWiFi#39] [Вопросы и ответы]
Добрый вечер! добавил полос в матрицу, прошился С ОПЦИЕЙ В МЕНЮ "ИНСТРУМЕНТЫ" - "Erase Flash" - "All Flash Contents" для ESP8266, в веб морде размер матрицы тоже увеличил, но по факту размер остался прежним, менял местами ленту добавлял другие куски, но ничего не получилось( подскажите куда копать?

**Comment by vvip-68** [comment]
Зайти на вкладку "Настройки" - "Оборудование" и прописать там реальное количество диодов, подключенное к пину
Кто б еще в вебморде, где задаются размеры матрицы, читал предупреждение, что это нужно сделать...

**Reply by kovalserg** [reply]
я прочитал))) найти не могу вкладку оборудование((( ищуу

**Reply by kovalserg** [reply]
нашелл)))))))
